express-session/express-session.d.ts

```diff
@@ -53,20 +53,39 @@ declare module "express-session" {
       unset?: string;
     }
 
-    export interface Store {
+    export interface IBaseStore {
```

Comment:
please remove `I` prefix from interface.
from http://www.typescriptlang.org/Handbook#writing-dts-files 

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.